### ISSUE TITLE
Adaptação do Name that Color > NodeJS

### ISSUE THREAD

**Issue by LFeh** [frontendbr/open-source#11]
## Nome
Name that Color > NodeJS

## Objetivo
Adaptar a [lib do Name that Color](http://chir.ag/projects/ntc/ntc.js) em NodeJS para fácil integração com Stylus e etc.

## Skills necessários
- JavaScript

## Projetos similares
http://chir.ag/projects/name-that-color/#6195ED

O que acham @frontendbr/admins?

**Comment by eduardojmatos** [comment]
Acho sensacional @LFeh =D

Mas acho que já existe um projeto assim cara: https://github.com/zeke/color-namer

**Comment by LFeh** [comment]
Porra, ,melhor ainda.

Analisando esse projeto, acha que ele pode fornecer suporte para Stylus? Compensa abrir uma issue lá pra discutir isso?

**Comment by eduardojmatos** [comment]
É, daí já não sei... E se fosse um mixin hein? Já ajudaria não?

**Comment by LFeh** [comment]
Um mixin com todas as variações do site né? Uhmmmm.... é uma boa ideia.

**Comment by eduardojmatos** [comment]
Exatamente! Ou na real, replicar essa lógica do JavaScript pro mixin. Acho
que ficaria foda =D

**Comment by LFeh** [comment]
Só que um mixin não conseguiria ter um sistema de aproximação de valores certo? Porque essa é a ideia, nem sempre vou ter o valor exato.

**Comment by eduardojmatos** [comment]
Mas o script tenta chegar nessa aproximação não?
